source/extensions/tracers/opentelemetry/opentelemetry_tracer_impl.h

```diff
@@ -8,8 +8,8 @@
 #include "source/common/common/logger.h"
 #include "source/common/singleton/const_singleton.h"
 #include "source/extensions/tracers/common/factory_base.h"
-#include "source/extensions/tracers/opentelemetry/grpc_trace_exporter.h"
-#include "source/extensions/tracers/opentelemetry/tracer.h"
+
+#include "tracer.h"
```

Comment:
please use the full path

Reply:
Done in: [2a5d544](https://github.com/envoyproxy/envoy/pull/29207/commits/2a5d5448406e99fbfaecda651e0d176eda96383a)